cli/webfinger.ts

```diff
@@ -17,6 +17,11 @@ export const command = new Command()
     "-p, --allow-private-address",
     "Allow private IP addresses in the URL.",
   )
+  .option(
+    "--max-redirection <maxRedirection:number>",
```

Comment:
[Cliffy provides an `integer` type.](https://cliffy.io/docs@v1.0.0-rc.7/command/types#built-in-types)

Reply:
I'll reflect this!